lib/tenstorrent/bh_arc/Kconfig

```
@@ -64,3 +64,12 @@ config TT_BH_ARC_SYSINIT
 	  Initialize blackhole firmware and hardware prior to starting the application.
 
 	  This allows our library drivers to work better with Zephyr's driver model.
+
+config TT_FWTABLE_IN_SPI
```

I think this should probably be changed to be disabled by default, which is more the norm in Zephyr.
```suggestion
config TT_FWTABLE_MOCK
```